src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceProvider.cs

```diff
@@ -37,6 +37,7 @@ public sealed class ServiceProvider : IServiceProvider, IKeyedServiceProvider, I
         internal static bool VerifyOpenGenericServiceTrimmability { get; } =
```

Comment:
Is this missing from src/libraries/Microsoft.Extensions.DependencyInjection/src/ILLink/ILLink.Substitutions.xml?

```suggestion
        [FeatureSwitchDefinition("Microsoft.Extensions.DependencyInjection.VerifyOpenGenericServiceTrimmability")]
        internal static bool VerifyOpenGenericServiceTrimmability { get; } =
```